src/products/dtos/product.dto.ts

```diff
@@ -0,0 +1,21 @@
+export class ProductDTO {
```

Comment:
Yes but I remove the DTO for now because I'm not currently requesting information. Just returning it using entities. On future, when create a DTO, I will use `class-validator` for sure